fuzz/zpay32/encode.go

```diff
@@ -0,0 +1,44 @@
+// +build gofuzz
+
+package zpay32fuzz
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/btcsuite/btcd/chaincfg"
+
+	"github.com/lightningnetwork/lnd/zpay32"
+)
+
+// Fuzz_encode is used by go-fuzz.
+func Fuzz_encode(data []byte) int {
+	inv, err := zpay32.Decode(string(data), &chaincfg.TestNet3Params)
+	if err != nil {
+		return 1
+	}
```

Comment:
Should we also sanity check here as in the `Fuzz_decode` test?

Reply:
Yeah couldn't hurt.